### DISCUSSION TITLE
Understanding of "End of Life" field in the release management

### DISCUSSION THREAD

**Discussion by enthunilu** [zephyrproject-rtos/zephyr#78173] [General] [answered]
Hi All,

This is general query on understanding "End of Life" field in the Milestone available in the release note management.
https://github.com/zephyrproject-rtos/zephyr/wiki/Release-Management

I see, current released Zephyr RTOS version is 3.7 and it says End of Life as 2027/01/26. 

However, at the moment I am using Zephyr RTOS v3.6 and it has EOL 2024/11/29.

Here, just wanted to get confirmation on this EOL means, after the mentioned time we will not be getting any support on the given version of RTOS. And recommended is to go for higher version of Zephyr RTOS.

Is my understanding correct ?

Thanks in advance!!

**Comment by nordicjm** [comment, marked as answer]
No support, no more commits or bug fixes pushed to that branch and would need to update, yes.

**Reply by enthunilu** [reply]
Thanks for the clarification. :-)
